### ISSUE TITLE
weird skaled crash at start if getting eth_sendRawTransaction and/or filter subscription

### ISSUE THREAD

**Issue by cstrangedk** [skalenetwork/skaled#412]
```
sudo ~/skaled-debug/create_btrfs.sh; sudo SKTEST_EXE=~/skaled/build-debug-release/skaled/skaled NO_ULIMIT_CHECK=1 DATA_DIR=btrfs pytest-3 -s 'test_node_rotation.py::test_late_join'
```

If uncomment ch.transaction() in test_late_join - strange crash on all nodes (sometimes in different places)

**Comment by cstrangedk** [comment]
➤ Dima Litvinov commented:

https://github.com/skalenetwork/skale-node-tests/blob/bls-again/test_node_rotation.py#L316 ( https://github.com/skalenetwork/skale-node-tests/blob/bls-again/test_node_rotation.py#L316 )

**Comment by cstrangedk** [comment]
➤ Ganna Kulikova commented:

Ready for triage

**Comment by kladkogex** [comment]
Close or fix for 2.1
